api/pkg/s3/routbucketget.go

```diff
@@ -27,8 +27,8 @@ func (s *APIService) RouteBucketGet(request *restful.Request, response *restful.
 		s.BucketAclGet(request, response)
 	} else if IsQuery(request, "uploads") {
 		s.ListBucketUploadRecords(request, response)
-	} else if IsQuery(request, "versioning") {
-		//TODO
+	} else if IsQuery(request, "allversions") {
```

Comment:
Is this be compatible with aws s3? From https://docs.aws.amazon.com/AmazonS3/latest/dev/list-obj-version-enabled-bucket.html, the URL of listing all object versions in a bucket is supposed to be GET /?versions HTTP/1.1, while Retrieving All Versions of Only a Specific Object is supposed to be GET /?versions&prefix=objectName HTTP/1.1.